epsilons/main_online_experiments.py

```diff
@@ -49,24 +65,50 @@ def create_agent(sess, environment, summary_writer=None): #memory=None
                                 "random seed": i,
                                 "agent": FLAGS.agent,
                                 "environment": FLAGS.env,
-                                "epsilon": eps, 
-                                "varying": "epsilon",
+                                f'{FLAGS.exp}': eps, 
+                                "varying": FLAGS.exp,
                                 "online": True,
 
                             },
                             reinit=True)
             with run:
                 agent_name = agents[FLAGS.agent].__name__
 
-                LOG_PATH = os.path.join(os.path.join(f'{path}/{FLAGS.agent}/{FLAGS.env}_{eps}_online', f'test{i}'))
+                LOG_PATH = os.path.join(os.path.join(f'{path}/{FLAGS.agent}/{FLAGS.env}/{FLAGS.exp}_{eps}_online', f'test{i}'))
                 sys.path.append(path)
                 gin_file = f'Configs/{FLAGS.agent}_{FLAGS.env}.gin'
 
-                gin_bindings = [f"{agent_name}.seed={FLAGS.initial_seed}", f"create_optimizer.eps = {eps}"]
+                if FLAGS.exp == "epsilons":
```

Comment:
shouldn't this be FLAGS.experiment?

etc.

Reply:
an alternative is creting a helper function where you pass in the experiment type and it returns the bindings. this will make the code cleaner and easier to follow.